### ISSUE TITLE
v9-gui: Adding a new switch, the ip field has "default" as default value

### ISSUE THREAD

**Issue by lzammit** [inverse-inc/packetfence#4425]
On the old admin, the switch ip/Mac address/network range would be empty.

<img width="1157" alt="image" src="https://user-images.githubusercontent.com/5261214/57465846-70add700-724d-11e9-8881-df5a2f062f85.png">

**Comment by cgx** [comment]
We need to modify the OPTIONS. Reassigning to @jrouzierinverse.

```bash
$ /usr/local/pf/sbin/pfperl-api get -M OPTIONS /api/v1/config/switches | jq '.meta.id'
{
  "default": "default",
  "placeholder": null,
  "required": true,
  "type": "string"
}
```

**Comment by satkunas** [comment]
This is due to `meta`.

```json
{
    "meta": {
        "id": {
            "default": "default"
        }
    },
    "status": 200
}```

**Comment by jrouzierinverse** [comment]
fixed by 4601309ab1cdb6298e924e3e62d7761859216093
